core/templates/dev/head/pages/subtopic_viewer/subtopic_viewer.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'pages/base.html' %}
+
+{% block maintitle %}
+  Skill Page
+{% endblock maintitle %}
+
+{% block header_js %}
+  {{ super() }}
+{% endblock header_js %}
+
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span>
+        {{ topic_title }}
+      </span>
+      <span class="oppia-navbar-breadcrumb-separator" style="margin-left: 5px;"></span>
+      <span>
+        {{ subtopic_title }}
+      </span>
+    </li>
+  </ul>
+{% endblock navbar_breadcrumb %}
+
+
+{% block content %}
+  <div ng-controller="SubtopicViewer" style="position: relative;">
+    <div>
+      <div class="oppia-static-header">
+        <h1>{{ subtopic_title }}</h1>
```

Comment:
We should avoid using jinja templates. I think SubtopicViewer could provide with the title.

Reply:
Likewise above.

Reply:
done

Reply:
Hi @tjiang11 -- not done?

(Fixing just this line is fine though.)